playbook-website/app/helpers/application_helper.rb

```diff
@@ -137,7 +137,7 @@ def format_search_hash(kit)
   def search_list
     all_kits = []
     formatted_kits = []
-    MENU["kits"].each do |kit|
```

Comment:
`MENU["kits"]` is cached. Why are we doing this?

Reply:
@jasoncypret Clarification: Actually Nick is pushing up a commit momentarily that will utilize the MENU (cached one) under the hood of our aggregate_kits method. We did need to create a new method that does the correct grouping to support the sidebar use cases for rails, react, and swift since the structure of the menu.yml is now different. But again, inside of that new method, we will reference the cached version. So check again soon on this one!